### PR TITLE
[Mailer] Support for LIST-MANAGEMENT-OPTIONS for unsubscribe list

### PR DESCRIPTION
**Pull request by tfavareille**
| Q             | A
| ------------- | ---
| Branch?       | 7.1
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Issues        |
| License       | MIT

### CODE REVIEW

File: src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiAsyncAwsTransport.php [hunk 1]
```diff
@@ -98,6 +98,32 @@ protected function getRequest(SentMessage $message): SendEmailRequest
...
+            } else {
+                $input['ContactListName'] = $email->getHeaders()->get('X-SES-CONTACT-LIST-NAME')->getBodyAsString();
+                
+                if ($header = $email->getHeaders()->get('X-SES-TOPIC-NAME')) {
+                    $input['TopicName'] = $header->getBodyAsString();
+                }
+            }
+            
+            if ($input['ContactListName']) {
+                $request['ListManagementOptions'] = new ListManagementOptions($input);
```
Comment (OskarStark) [inline src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiAsyncAwsTransport.php:124]: The class does not exist

Comment (fabpot) [reply src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiAsyncAwsTransport.php:124]: @tfavareille Any news?

Comment (tucksaun) [reply src/Symfony/Component/Mailer/Bridge/Amazon/Transport/SesApiAsyncAwsTransport.php:124]: missing use for `AsyncAws\Ses\ValueObject\ListManagementOptions` (see https://github.com/async-aws/aws/blob/master/src/Service/Ses/src/ValueObject/ListManagementOptions.php) ?

**Review by OskarStark — CHANGES_REQUESTED** [review]
Please add tests, thanks

**Comment by fabpot** [comment]
Closing as there is no more activity.

**Pull request closed without merging**